dist/fabric.js

```diff
@@ -25973,12 +25974,6 @@ fabric.Image.filters.BaseFilter.fromObject = function(object, callback) {
       this.__skipDimension = true;
       this.callSuper('initialize', options);
 
-      if (this.placeholderMode && options.placeholderColor) {
```

Comment:
don't need this color switching, we can apply placeholder color directly during textbox rendering